docs/checkout.md

```diff
@@ -1,5 +1,37 @@
 # Checkout
 
+## Configuration
+
+You can customize the steps of checkout by creating your own. This is the default one :
+
+```$yaml
+# config/packages/sylius_gtm_enhanced_ecommerce.yaml
+
+sylius_gtm_enhanced_ecommerce:
+    features:
+        checkout:
+            steps:
+                1:
+                    -
+                        event: "click"
```

Comment:
Let's make the arguments alphabetic (event, option, selector) to have some coherence on that.